### PR TITLE
[pytest/ansible] Add support for multi-duts

### PR DESCRIPTION
**Pull request by tahmed-dev**
Adding support for multiple test device for SONiC testbed version 2

signed-off-by: Tamer Ahmed <tamer.ahmed@microsoft.com>

Fixes # (issue)

### Type of change

- [] Bug fix
- [x] Testbed and Framework(new/improvement)
- [] Test case(new/improvement)

#### How did you verify/test it?
runtest.sh on virtual test bed (kvm)

### CODE REVIEW

File: ansible/testbed-cli.sh [hunk 1]
```diff
@@ -55,37 +55,38 @@ function usage
...
+
+  testbed_name=${line_arr[1]}
+  topo=${line_arr[2]}
+  ptf_imagename=${line_arr[3]}
+  ptf=${line_arr[4]}
+  ptf_ip=${line_arr[5]}
+  server=${line_arr[6]}
+  vm_base=${line_arr[7]}
+  dut=${line_arr[8]//;/,}
+  dut=${dut//[\[\]]/}
```
Comment (yxieca) [inline ansible/testbed-cli.sh]: s/dut/duts/ ?

**Comment by yxieca** [comment]
Do you also need to change testbed.csv?

File: ansible/testbed-cli.sh (hunk 1 above)
Comment (tahmed-dev) [reply ansible/testbed-cli.sh]: OK. Will change it. This will require to touch up the whole script to as the $dut is referenced in multiple locations.

**Comment by tahmed-dev** [comment]
> Do you also need to change testbed.csv?

The current change does not require change being made to testbed.csv. the '[' ']' are stripped out and contents is converted to python list. The comma separate DUTs could be also used in Ansible yml file to limit execution to those specific DUTs (might not be important since we are moving away from Ansible)

However there were some touch up that I would like to make but would results in more changes such as:

1. changing 'dut' in the header line field to duts and this would require similar change to testbed.csv. 
2. Using semi colon as separator for fields while using comma as separator for duts (it is more cosmetic)

**Pull request merged**